ui/components/component-library/button-base/button-base.js

```diff
@@ -54,6 +55,7 @@ export const ButtonBase = ({
       display={DISPLAY.INLINE_FLEX}
       justifyContent={JUSTIFY_CONTENT.CENTER}
       alignItems={ALIGN_ITEMS.CENTER}
+      borderRadius={BORDER_RADIUS.PILL}
       {...props}
     >
       <Text
```

Comment:
This was a test to see if we can save on CSS and overwriting CSS. If we want to remove the border-radius from the ButtonLink component we can just use `borderRadius={null}` - it removes the Box class. So less extra CSS on the base component and less CSS to overwrite the base component on the variant